src/Core/src/ImageSources/UriImageSourceService/UriImageSourceService.Android.cs

```diff
@@ -13,6 +13,29 @@ namespace Microsoft.Maui
 {
 	public partial class UriImageSourceService
 	{
+		public override Task<bool> LoadDrawableAsync(IImageSource imageSource, Android.Widget.ImageView imageView, CancellationToken cancellationToken = default)
+		{
+			if (imageSource is IUriImageSource uriImageSource)
+			{
+				try
+				{
+					Glide
+						.With(imageView.Context)
+						.Load(uriImageSource.Uri.OriginalString)
+						.Into(imageView);
```

Comment:
I think this is the problem I never solved -- even back in Xamarin.Forms.

Here Glide will load the image based on the ImageView's size, and MAUI might want to size the ImageView based on the size of the image. So like a StackLayout or Grid Row/Column set to Auto, would have this image as 0x0.

Reply:
This seems to work though... I was testing this with some cases locally using .WaitForLayout() on that call which seems to wait until the android ImageView gets sized, so I think that avoids this?